server/views/applications/withdrawables/new.njk

```diff
@@ -19,6 +19,11 @@
     <div class="govuk-grid-row">
         <div class="govuk-grid-column-two-thirds">
             {% if withdrawables.length %}
+                <ul class="govuk-list govuk-list--bullet">
```

Comment:
We should check that the notes exist before creating the `ul` otherwise we will be adding an empty list to the markup